source/blog/043-import-is-deprecated.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "`@import` is Deprecated"
+author: Jennifer Thakar
+date: 2024-10-15 13:00:00 -8
+---
+
+Back in 2019, we [released the Sass module system], adding new `@use` and
+`@forward` rules to the language that were designed to replace `@import` and
+make stylesheets more maintainable and less error-prone. We can now announce
+that `@import` is officially deprecated as of Dart Sass 1.78.0.
+
+[released the Sass module system]: /blog/the-module-system-is-launched
+
+The module system ensures that it's easy for both developers and tooling to
+determine where a given Sass member is defined, adds namespacing to prevent the
+need to manually add long, awkward namespaces to names, and allows library
+authors to ensure their private helpers can't be accessed by downstream users.
+Additionally, since each module is only ever loaded once, depending on the same
+stylesheet multiple times no longer results in duplicated CSS.
+
+With 4.5 years since we released the module system and more than a year since
+we passed the 80% Dart Sass usage share threshold we set for starting this
+deprecation, we feel comfortable making this move. However, we understand that
+this is a big change to the language and not all users have been able to fully
+migrate off of `@import` so far.
+
+## Controlling Deprecation Warnings
+
+While users should treat this deprecation as a signal to migrate sooner rather
+than later, for anyone who wishes to silence deprecation warnings for
+`@import`, we recently added a new `--silence-deprecation` command line option.
+Just pass `--silence-deprecation=import` to Sass and you won't have to worry
+about the deprecation warnings while you complete the migration. Equivalent
+options exist in the JS and Dart APIs.
+
+On the other hand, users who have already migrated to the module system and
+wish to prevent any backsliding can use the `--fatal-deprecation=import` to
+treat any Sass `@import` rules as errors.
+
+## Timeline
+
+As previously announced, given the size of this change, we expect to wait at
+least a year after this deprecation before we remove `@import` from the
```

Comment:
Let's say "at least two years". I think we want to take this nice and slow.

Reply:
Done